__tests__/api-paths.test.js

```diff
@@ -108,3 +97,51 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+describe("GET /api/reviews/:review_id", () => {
+  test("status: 200 returns an array with review objects", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then((data) => {
+        const reviews = data.body.reviews;
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              title: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_id: 3,
+              review_img_url: expect.any(String),
+              category: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              review_body: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("should return only the required review", () => {
+    return request(app)
+      .get("/api/reviews/4")
+      .expect(200)
+      .then((data) => {
+        const result = data.body.reviews;
+        expect(result.length).toBe(1);
+      });
+  });
+});
+
+describe("error handeling", () => {
+  test("status:404 when serching for an incorret path", () => {
+    return request(app)
+      .get("/api/not_a_path")
+      .expect(404)
+      .then((data) => {
+        expect(data.body.msg).toBe("Path not found");
+      });
+  });
+  test("status:400 when serching for a missing id in reviews", () => {
+    return request(app).get("/api/reviews/92").expect(400);
```

Comment:
This is a good test, however when it's a `review_id` that is not found in the database we would expect to return a 404 - we then need another test for if the `review_id` is completely invalid ( `review_id = banana` ) - this will break the SQL query and psql will return an error for you on code `22P02` - which we should handle as a status `400`